src/modules/registrypreview/RegistryPreviewUI/RegistryPreviewXAML/MainWindow.xaml

```diff
@@ -114,9 +116,11 @@
                     <AppBarButton
                     x:Name="saveButton"
                     x:Uid="SaveButton"
-                    Icon="Save"
```

Comment:
Does this work the same as a **SymbolIcon**? If so, using the glyph code does play it safe. I know that SymbolIcon element is bad, but not sure about AppBar.

Reply:
Completely agree, if the fallback logic for all the OS's and all the languages worked.  It started using the Icon property and only went the more pedantic route after the bug for Win 10 in traditional Chinese was reported.

Reply:
What I was thinking of was more the glyphs it uses; all in E1** range